flirtsch/README.md

```diff
@@ -1,3 +1,5 @@
 # Schedule files to use with FSL FLIRT
 
-N.B. schedule_TxTy.sch should stay untouched as it is referenced in Tanguy's paper.
\ No newline at end of file
+N.B. This folder can't be moved or changed, because the path to `flirtsch/schedule_TxTy.sch` is hardcoded in the following article:
+
+* [Duval, Tanguy et al. “In vivo mapping of human spinal cord microstructure at 300mT/m.” NeuroImage vol. 118 (2015): 494-507. doi:10.1016/j.neuroimage.2015.06.038](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4562035/#idm139707147392480title)
```

Comment:
Note: I wanted to also move `flirtsch` into `contrib`, but I understand that the path to `schedule_TxTy.sch` is hardcoded in [Tanguy's NeuroImage article](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4562035/#idm139707147392480title), so it can't really be touched.

Reply:
> Note: I wanted to also move `flirtsch` into `contrib`, but I understand that the path to `schedule_TxTy.sch` is hardcoded in [Tanguy's NeuroImage article](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4562035/#idm139707147392480title), so it can't really be touched.

@alexfoias can you please contact the publisher to see if we can update that link too?

Reply:
If we can, then moving it into a dedicated repo (rather than `contrib`) might actually be the best long-term solution, now that I think about it.

Reply:
Okay, I missed the ping. I have contacted the publisher. Usually in the past in wasn't possible to change the link. Maybe we will be more lucky with this particular publisher.

Reply:
@joshuacwnewton could you please point me to the link that we want to replace ? thanks.

Reply:
Assuming the repository name, `README.md`, etc. look good, here is a link to the repository I've created:

https://github.com/neuropoly/neuroimage-fsl-flirt-schedule-files

Reply:
So we want to change link 1  https://github.com/neuropoly/spinalcordtoolbox/blob/master/flirtsch/schedule_TxTy.sch --> https://github.com/neuropoly/neuroimage-fsl-flirt-schedule-files in article https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4562035/#!po=72.5000  ?

Reply:
That was my thinking, yes. :slightly_smiling_face:

Reply:
Thanks. I sent the info to the publisher.

Reply:
We cannot do the change. Here are the details:
I wrote this:
```
We want to change the URL from link 1 to https://github.com/neuropoly/neuroimage-fsl-flirt-schedule-files#readme because the contents of the URL listed in the article was migrated there.

Would it be possible to fix it ?
```
Their answer:

```
Apologies as we are not able to make changes on this site  (github.com) as this is not managed by Elsevier. It is the first time I have seen this webpage.
Should you wish to make changes/updates on this webpage, I would suggest to contact them directly. https://support.github.com/?tags=dotcom-footer
I hope this helps to clarify.
```

Reply:
Reading their response, I think they misunderstood the request. :sweat_smile: 

We aren't asking Elsevier to make changes to github.com, because we have already made those changes ourselves. :stuck_out_tongue_closed_eyes:

Reply:
@joshuacwnewton is right-- Alex can you pls rephrase your question to elsevier-- make sure they understand-- pls have your question reviewed by someone in the team-- thx

Reply:
My initial email was: 

```
We have migrated the content from link 1 https://github.com/neuropoly/spinalcordtoolbox/blob/master/flirtsch/schedule_TxTy.sch to the new location https://github.com/neuropoly/neuroimage-fsl-flirt-schedule-files.

Would it be possible to change the link in the article ?
```

Reply:
By the way, to avoid this in the future, you can link to a tag, e.g. https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/v3.0.4/documentation/logo_sct.png

Sadly this doesn't work for this particular file, because the `flirtsch/` folder was removed from our `release` branch at some point and all of our tags are made out of our `release` branch. But the principle is sound.

Reply:
> My initial email was:
> 
> ```
> We have migrated the content from link 1 https://github.com/neuropoly/spinalcordtoolbox/blob/master/flirtsch/schedule_TxTy.sch to the new location https://github.com/neuropoly/neuroimage-fsl-flirt-schedule-files.
> 
> Would it be possible to change the link in the article ?
> ```

@alexfoias Has there been any response from the publishers to this updated email you sent?

If we're still waiting, then I'm thinking that it might be best to merge this now, and just open an issue for the `flirtsch` folder so we can follow up later.

Reply:
@joshuacwnewton that sounds good. I think is better to merge now.

Reply:
==> https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3559